datahub-web-react/src/app/entity/schemaField/SchemaFieldPropertiesEntity.tsx

```diff
@@ -33,9 +34,24 @@ export class SchemaFieldPropertiesEntity implements Entity<SchemaFieldEntity> {
     // Currently unused.
     renderProfile = (_: string) => <></>;
 
-    renderPreview = (previewType: PreviewType, data: SchemaFieldEntity) => (
-        <Preview previewType={previewType} datasetUrn={data.parent.urn} name={data.fieldPath} />
-    );
+    renderPreview = (previewType: PreviewType, data: SchemaFieldEntity) => {
+        const parent = data.parent as Dataset;
+        const properties = parent.properties as EntityNameProperties;
```

Comment:
could we not just use the properties on `Dataset` here (`DatasetProperties`)? they already have `name` and `qualifiedName` in the correct types i believe right?

Reply:
`DatasetProperties` require `lastModified` as a mandatory field which in our case was missing, hence we defined a new type `EntityNameProperties` containing only name and `qualifiedName`.

However, as per the [last comment](https://github.com/datahub-project/datahub/pull/11585#discussion_r1815344337), now we are passing `parentDataset` object, so we don't require type to be defined for `properties`.